test/cookbooks/php_test/metadata.rb

```diff
@@ -3,12 +3,14 @@
 maintainer_email 'chef@osuosl.org'
 license          'Apache-2.0'
 chef_version     '>= 12.18'
-issues_url       'https://github.com/osuosl-cookbooks/php_ini-test/issues'
-source_url       'https://github.com/osuosl-cookbooks/php_ini-test'
-description      'Installs/Configures php_ini-test'
+issues_url       'https://github.com/osuosl-cookbooks/osl-php/issues'
+source_url       'https://github.com/osuosl-cookbooks/osl-php'
+description      'Installs/Configures php_test'
 version          '0.1.0'
 
 depends          'osl-php'
 depends          'osl-apache'
+depends          'yum-ius'
+depends          'yum-osuosl'
```

Comment:
Unless you're directly using recipes or resources from these cookbooks, please remove them as they will get pulled in from osl-php.